### PR TITLE
Update QStash Python SDK docs

### PR DESCRIPTION
**Pull request by mdumandag**
We have rewritten the SDK, so this PR updates the places where we gave SDK code snippets for it.

It should be merged after we release the new version of the SDK.

### CODE REVIEW

**Comment by CahidArda** [comment]
should rename this file as well I think https://github.com/upstash/docs/blob/0b17486fb0cc4b882d8a6c51195f46f92a5eae15/qstash/features/topics.mdx

**Comment by mdumandag** [comment]
Let's leave that to a later, when we update the UI etc. I intentionally didn't change any other part than the Python related parts where we renamed topic to url group.

**Comment by CahidArda** [comment]
> Let's leave that to a later, when we update the UI etc. I intentionally didn't change any other part than the Python related parts where we renamed topic to url group.

we updated the console btw, qstash-js is also updated.

we should update docs/examples to use url group instead of topic. I will update them

**Comment by CahidArda** [comment]
I have completed the changes in Qstash and Qstash JS.

there are already too many changes in this PR, I feel like we can go ahead with this PR and complete rest of the topic -> URL group changes in another PR.

File: oss/sdks/py/qstash/examples/url-groups.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (CahidArda) [inline oss/sdks/py/qstash/examples/url-groups.mdx:1]: can we create py/qstash/examples/queues.mdx file and examples for queues?

**Review by CahidArda — CHANGES_REQUESTED** [review]
can we add examples for queues in

**Review by CahidArda — APPROVED** [review]

**Pull request merged**